.github/workflows/tests-pa11y.yml

```
@@ -35,4 +35,31 @@ jobs:
         run: |
           npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
           "npx http-server dist --port 6006 --silent" \
-          "npx wait-on tcp:6006 && npm run test:pa11y"
\ No newline at end of file
+          "npx wait-on tcp:6006 && npm run test:pa11y" || true
+
+      - name: Archive pa11y report
+        uses: actions/upload-artifact@v3
+        with:
+          name: pa11y-report
+          path: ${{ github.workspace }}/pa11y-ci-report
+          
+  publish-results:
+    name: Publish pa11y Report
+    needs: run-tests
+    if: always()
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Deploy Artifacts
```

I don't think that we need a "whole job" for this, we can `if: always()` a step

```suggestion
      - name: Deploy Artifacts
        if: always()
```